arangod/IResearch/IResearchLinkHelper.cpp

```diff
@@ -662,6 +657,19 @@ namespace iresearch {
   return nullptr;
 }
 
+#ifdef USE_ENTERPRISE
+/*static*/ void IResearchLinkHelper::realNameFromSmartName(std::string& possiblySmartName) {
```

Comment:
this must be a part of enterprise repo :-) afair @mchacki has already introduced the similar method somewhere